### PR TITLE
Added Button component and storybook addons

### PR DESCRIPTION
**Pull request by ShreyaThapa**
- Added Button Component
- Added Button story
- Added Button test
- Added Storybook addons
    - for centering component - @storybook/addon-centered/react
    - for adding dark background - @storybook/addon-backgrounds

### CODE REVIEW

File: stories/Button.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import centered from "@storybook/addon-centered/react";
+import Button from "../components/base/Button";
+
+storiesOf("Button", module)
```
Comment (sausman) [inline stories/Button.stories.tsx]: Let's move this file to the `stories/base` directory and change this line to `storiesOf("base|Button", module) `

File: components/base/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  return (
+    <ButtonStyle type="button" className={`${size} ${variant}`}>
+      <SpanStyle className={`${size}`}>{text}</SpanStyle>
+    </ButtonStyle>
+  );
+}
+
+const ButtonStyle = styled.button`
+  color: ${WHITE_PRIMARY};
+  font-family: "Poppins", sans-serif;
```
Comment (sausman) [inline components/base/Button.tsx]: Let's import `POPPINS` from `typography.ts` here

File: stories/Button.stories.tsx (hunk 1 above)
Comment (ShreyaThapa) [reply stories/Button.stories.tsx]: Sounds good! I'll get those changes in.

File: components/base/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  GREY_4,
+  GREY_9,
+  GREY_5,
+  PURPLE_DARK,
+} from "../colors";
+import { BOX_SHADOW_6 } from "../shadowDepths";
+
+type Props = {
+  text: string;
+  size: string;
```
Comment (sausman) [inline components/base/Button.tsx]: For `size` and `variant` we could use [union types](https://www.typescriptlang.org/docs/handbook/advanced-types.html#union-types) to specify the accepted values. For example:

`size: "tiny" | "small" | "standard" | "large";`

Comment (ShreyaThapa) [reply components/base/Button.tsx]: Thanks for catching that! 👍

Comment (ShreyaThapa) [reply components/base/Button.tsx]: This [commit](https://github.com/Dwolla/dev-portal/pull/15/commits/7d47ff01d96b5204451ad2cd7f5ec6d82aa5b3c7) should fix that.

**Pull request merged**